### PR TITLE
Change Finance::Quote module install from VBS/CMD to Windows Powershell

### PR DESCRIPTION
**Pull request by ericmoon**
Removes VBS and command.exe scripts from extra_dist and replaces with Windows Powershell (note, Windows Powershell, not Powershell, which is different).

Also changes the Inno Setup scripts to have the Start Menu point to the powershell script. 

I was able to test all of the functionality except the install of the Start Menu shortcut as I don't have that build environment setup yet.

### CODE REVIEW

File: extra_dist/install-fq-mods.ps1 [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+function chkPerlVer {
+    
+    Write-Host "`n * Checking Perl version`n"
+   
+    perl -e "exit(int($]));"
+    $perlMajor = $LastExitCode
+    perl -e "`$ver=1000*sprintf('%0.3f', $]); exit(int(`$ver)-5000);"
+    $perlMinor = $LastExitCode
+
+    if ($perlMajor -eq 5) {
```
Comment (jralls) [inline extra_dist/install-fq-mods.ps1:64]: Perl 5.0 was released in 1993 and Perl 6 is Raku. The likelihood of this failing is nil so please remove it.

File: extra_dist/install-fq-mods.ps1 [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        if ($perlminor -lt $perlVersion) {
+            Write-Host "`n Found perl version $perlMajor.$perlMinor, but GnuCash requires at least version 5.8."
+            Write-Host "Please manually install version 5.8 or above of"
```
Comment (jralls) [inline extra_dist/install-fq-mods.ps1:67]: Please use `$perlversion` in the string instead of hard-coding "5.8".

**Review by jralls — CHANGES_REQUESTED** [review]
The merge commit was helpful for pointing out to me that I hadn't pushed the commit to put the AQBanking beta back after the release on Sunday, but we really can't have merges in PRs. Please rebase on `https://github.com/Gnucash/gnucash-on-windows master` and force-push that to here.

File: inno_setup/gnucash-mingw64.iss [hunk 3]
```diff
@@ -46,7 +46,7 @@ Name: "{group}\GnuCash"; Filename: "{app}\bin\@PACKAGE@.exe"; WorkingDir: "{code
 Name: "{group}\{cm:IconName_README}"; Filename: "{app}\doc\@PACKAGE@\{cm:IconFilename_README}"; Comment: "{cm:IconComment_README}"; Tasks: menuicon
 Name: "{group}\{cm:IconName_FAQ}"; Filename: "http://wiki.gnucash.org/wiki/FAQ"; Tasks: menuicon
 Name: "{group}\{cm:IconName_Bugzilla}"; Filename: "https://bugs.gnucash.org/enter_bug.cgi?product=GnuCash"; Tasks: menuicon
-Name: "{group}\{cm:IconName_InstallFQ}"; Filename: "{app}\bin\install-fq-mods.cmd"; WorkingDir: "{app}\bin"; Comment: "{cm:IconComment_InstallFQ}"; Tasks: menuicon
+Name: "{group}\{cm:IconName_InstallFQ}"; Filename: "powershell.exe -ExecutionPolicy Bypass -File ""{app}\bin\install-fq-mods.ps1"""; WorkingDir: "{app}\bin"; Comment: "{cm:IconComment_InstallFQ}"; Tasks: menuicon
```
Comment (jralls) [inline inno_setup/gnucash-mingw64.iss:49]:
```
Running Inno Setup to create C:\gcdev64\gnucash\releases\gnucash-5.8.setup.exe.
Error on line 49 in C:\gcdev64\gnucash\releases\gnucash.iss: Parameter "Filename" cannot include quotes (")
Compile aborted.
```
Taking out the quotes around `{app}\bin\install-fq-mods.ps1` lets it compile but after installing the package the Install Online Price Retrieval shortcut's target is empty so it doesn't show up in the Start menu.

Comment (jralls) [reply inno_setup/gnucash-mingw64.iss:49]: This works:
```diff
--- a/inno_setup/gnucash-mingw64.iss
+++ b/inno_setup/gnucash-mingw64.iss
@@ -46,7 +46,7 @@ Name: "{group}\GnuCash"; Filename: "{app}\bin\@PACKAGE@.exe"; WorkingDir: "{code
 Name: "{group}\{cm:IconName_README}"; Filename: "{app}\doc\@PACKAGE@\{cm:IconFilename_README}"; Comment: "{cm:IconComment_README}"; Tasks: menuicon
 Name: "{group}\{cm:IconName_FAQ}"; Filename: "http://wiki.gnucash.org/wiki/FAQ"; Tasks: menuicon
 Name: "{group}\{cm:IconName_Bugzilla}"; Filename: "https://bugs.gnucash.org/enter_bug.cgi?product=GnuCash"; Tasks: menuicon
-Name: "{group}\{cm:IconName_InstallFQ}"; Filename: "powershell.exe -ExecutionPolicy Bypass -File ""{app}\bin\install-fq-mods.ps1"""; WorkingDir: "{app}\bin"; Comment: "{cm:IconComment_InstallFQ}"; Tasks: menuicon
+Name: "{group}\{cm:IconName_InstallFQ}"; Filename: "{syswow64}/WindowsPowershell/v1.0/powershell.exe"; Parameters: "-ExecutionPolicy Bypass -File ""{app}\bin\install-fq-mods.ps1"""; WorkingDir: "{app}\bin"; Comment: "{cm:IconComment_InstallFQ}"; Tasks: menuicon
 Name: "{group}\{cm:IconName_Uninstall}"; Filename: "{uninstallexe}"; Comment: "{cm:IconComment_Uninstall}"; Tasks: menuicon

 Name: "{commondesktop}\GnuCash"; Filename: "{app}\bin\@PACKAGE@.exe"; WorkingDir: "{code:GetDocPath}"; Comment: "{cm:IconComment_GnuCash}"; IconFilename: "{app}\share\@PACKAGE@\pixmaps\gnucash-icon.ico"; Tasks: desktopicon

```

**Comment by ericmoon** [comment]
Rebased and force-pushed.  It's been a while, so let me know if I screwed anything up.

File: extra_dist/install-fq-mods.ps1 (hunk 1 above)
Comment (jralls) [reply extra_dist/install-fq-mods.ps1:64]: It's still here.

File: extra_dist/install-fq-mods.ps1 [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+   
+    perl -e "exit(int($]));"
+    $perlMajor = $LastExitCode
+    perl -e "`$ver=1000*sprintf('%0.3f', $]); exit(int(`$ver)-5000);"
+    $perlMinor = $LastExitCode
+
+    if ($perlMajor -eq 5) {
+        if ($perlminor -lt $perlVersion) {
+            Write-Host "`n Found perl version $perlMajor.$perlMinor, but GnuCash requires at least version 5.$perlVersion."
+            Write-Host "Please manually install version 5.8 or above of"
```
Comment (jralls) [inline extra_dist/install-fq-mods.ps1:67]: Another place for `5.$perlVersion`.

File: extra_dist/install-fq-mods.ps1 [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+        $done = "(done) "
+    }
+    else {
+        $done = ""
+    }
+    Write-Host "  ***    - setting the environment variable ALPHAVANTAGE_API_KEY $done or"
+    Write-Host "  ***    - starting GnuCash and adding the Alpha Vantage api key in"
+    Write-Host "  ***        Edit->Preferences->Online Quotes"
+    Write-Host "  ***`n"
+    pause
```
Comment (jralls) [inline extra_dist/install-fq-mods.ps1:104]: New item, I should have noticed this before.  First it doesn't really make sense to test `ALPHAVANTAGE_API_KEY` in the middle, but more significantly it's not necessary to set the environment variable at all. Users can set a preference in GnuCash if they need Alphavantage, but that need is much less than it was. I'm inclined towards removing this and leaving it for GnuCash to deal with at runtime.

**Review by jralls — CHANGES_REQUESTED** [review]

**Comment by jralls** [comment]
You went a bit far with bumping the perl version, 5.38.2.2 isn't available for 32-bit. I'm testing now but my VM is really slow and it's taking forever. Unfortunately I'll be away from this keyboard for a couple of days so I may not be able to push the merge until Thursday.

**Comment by jralls** [comment]
Thanks!

**Pull request merged**